From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: domicile CI <ci@domicile.invalid>
Date: Tue, 6 Oct 2026 00:00:00 +0000
Subject: [PATCH] domicile: the members the system libraries replaced go

Battery, backlight, audio and the application search are libraries on
the shell's system calls now, so `searchApps()` and the mixer's
interfaces -- DomicileAppSearch, DomicileDesktopEntry, DomicileBookmark,
DomicileAudioCard, DomicileAudioChoice, DomicileAudioDevice,
DomicileAudioStream, DomicileAudioLevel and DomicileAudioLevelsEvent --
leave the module, and their IDL and generated bindings leave the lists
here.

Verified: not built. This repository cannot build Chromium; the engine
job builds it on `crux`.
---
 ...link/renderer/bindings/generated_in_modules.gni | 20 --------------------
 ...arty/blink/renderer/bindings/idl_in_modules.gni | 10 ----------
 2 files changed, 30 deletions(-)

diff --git a/third_party/blink/renderer/bindings/generated_in_modules.gni b/third_party/blink/renderer/bindings/generated_in_modules.gni
--- a/third_party/blink/renderer/bindings/generated_in_modules.gni
+++ b/third_party/blink/renderer/bindings/generated_in_modules.gni
@@ -318,10 +318,6 @@
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_file_search.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_file_preview.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_file_preview.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_app_search.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_app_search.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_levels_event_init.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_levels_event_init.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_shortcut.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_shortcut.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_shortcut_event_init.cc",
@@ -2389,26 +2385,10 @@
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_clipboard_entry.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_extension.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_extension.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_desktop_entry.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_desktop_entry.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_bookmark.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_bookmark.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_browser_window.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_browser_window.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_display.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_display.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_card.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_card.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_choice.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_choice.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_device.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_device.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_stream.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_stream.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_level.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_level.h",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_levels_event.cc",
-  "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_audio_levels_event.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_shortcut_event.cc",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_shortcut_event.h",
   "$root_gen_dir/third_party/blink/renderer/bindings/modules/v8/v8_domicile_tray_item.cc",
diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -285,19 +285,9 @@
   "//third_party/blink/renderer/modules/domicile/domicile_extension.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_file_search.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_file_preview.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_app_search.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_desktop_entry.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_bookmark.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_browser_window.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_host.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_display.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_card.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_choice.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_device.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_stream.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_level.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_levels_event.idl",
-  "//third_party/blink/renderer/modules/domicile/domicile_audio_levels_event_init.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_shortcut.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_shortcut_event.idl",
   "//third_party/blink/renderer/modules/domicile/domicile_shortcut_event_init.idl",
-- 
2.43.0

